apps/core/schema.py

```diff
@@ -49,9 +53,363 @@ class Meta:
         filterset_class = DatasetFilter
 
 
+class KeyValueType(graphene.ObjectType):
+    key = graphene.String()
+    label = graphene.String()
+
+
+class TablePropertiesType(graphene.ObjectType):
+    headers = graphene.List(KeyValueType)
+    languages = graphene.List(KeyValueType)
+    time_zones = graphene.List(KeyValueType)
+
+    def resolve_headers(self, info):
+        data = [
+            {
+                "key": "1",
+                "label": "One",
+            },
+            {
+                "key": "1",
+                "label": "One",
+            },
+            {
+                "key": "1",
+                "label": "One",
+            },
+            {
+                "key": "1",
+                "label": "One",
+            },
+            {
+                "key": "1",
+                "label": "One",
+            },
+        ]
+        output = []
+        for d in data:
+            output.append(
+                KeyValueType(
+                    key=d["key"],
+                    label=d["label"],
+                )
+            )
+        return output
+
+    def resolve_languages(self, info):
+        data = [
```

Comment:
Also, let's have a separate module, something like `table_properties` and have all the constants there.